changelog/README.md

```diff
@@ -1,5 +1,117 @@
 # Changelog
 
+## 12th June 2023
+
+* Added operation [Get all product categories](../operations/productcategories.md#get-all-product-categories) to retrieve product categories.
+
+* Enabled multi-property access token for following endpoints:
```

Comment:
Just need to make sure we are consistent with terminology. So we might say "Enabled multi-property working for the following operations:" or "Modified the following operations to support Portfolio Access Tokens:" (so 'operations' not 'endpoints', and 'Portfolio Access Token' not 'multi-property token').